core/src/main/java/org/elasticsearch/index/query/FieldMaskingSpanQueryBuilder.java

```diff
@@ -30,17 +35,37 @@
 
     private final SpanQueryBuilder queryBuilder;
 
-    private final String field;
+    private final String fieldName;
 
-    private float boost = -1;
+    private float boost = 1.0f;
 
     private String queryName;
 
     static final FieldMaskingSpanQueryBuilder PROTOTYPE = new FieldMaskingSpanQueryBuilder(null, null);
 
-    public FieldMaskingSpanQueryBuilder(SpanQueryBuilder queryBuilder, String field) {
+    /**
+     * Constructs a new {@link FieldMaskingSpanQueryBuilder} given an inner {@link SpanQueryBuilder} for
+     * a given field
+     * @param queryBuilder inner {@link SpanQueryBuilder}
+     * @param fieldName the field name
+     */
+    public FieldMaskingSpanQueryBuilder(SpanQueryBuilder queryBuilder, String fieldName) {
         this.queryBuilder = queryBuilder;
-        this.field = field;
+        this.fieldName = fieldName;
+    }
+
+    /**
+     * @return the field name for this query
+     */
+    public String fieldName() {
+        return this.fieldName;
+    }
+
+    /**
+     * @return the inner {@link QueryBuilder}
+     */
+    public SpanQueryBuilder innerQuery() {
```

Comment:
what naming convention do we want to follow here? I saw around some innerQueryBuider(), here innerQuery(), maybe we even want to drop the inner prefix... let's decide for one of the options and go for it everywhere.